http/bucket_service_test.go

```diff
@@ -456,7 +456,7 @@ func TestService_handlePostBucket(t *testing.T) {
 				},
 			},
 			wants: wants{
-				statusCode: http.StatusBadRequest,
+				statusCode: http.StatusUnprocessableEntity,
```

Comment:
If this is a breaking change, we should surface this to monitoring and document it in our swagger.

Reply:
good catch, I'll circle the wagons on that right now

Reply:
lol none of the error cases are in the swagger docs 🤦‍♀

Reply:
or maybe the status code just doesn't affect it 🤔, clear as mud atm

Reply:
one other note, this shouldn't' be considered a breaking change. there was nothing stopping us from returning a 422 earlier. Hopefully it was, hard to track it down, goes through 4 layers of indirection before finding an originating error. Fwiw, the status code it is changed ot is the correct one, a 400 is more for unmarshaling failing to decode/read something. 422 typically outlines validation failure. Milealge may vary 🤷‍♂

Reply:
We probably overuse 400 responses in the API as kind of a catch-all default error response (and it's, in fact, our `default` for responses in the swagger).

I'd consider this to be breaking in that it's a change in behavior. If javascript in the UI is looking for a 400 response instead of a 400-level response, the UI would break. Pretty minor in this case, and suggests that the UI is brittle if it can't handle all failure situations, but it's a change nonetheless.

Not blocking though - I can surface this information to other people. Would be worthwhile to add the 422 response to the swagger though to make sure it's documented.

Reply:
good points on all accounts, I pushed up the swagger changes 👍